packages/crossbuild/src/typings/types.ts

```diff
@@ -13,6 +13,9 @@ export interface Config {
 	name: string
 	/** Paths that the bot should search for components under */
 	componentPaths: string[]
+	/** The prefix that the bot should use. If none is specified, it will default to mentioning the bot */
```

Comment:
```diff
- /** The prefix that the bot should use. If none is specified, it will default to mentioning the bot */
+ /** The prefix that the bot listens for. If none is specified, the prefix will default to mentioning the bot */
```